Firmata.cpp

```diff
@@ -398,7 +437,11 @@ void FirmataClass::attach(uint8_t command, systemCallbackFunction newFunction)
  */
 void FirmataClass::attach(uint8_t command, stringCallbackFunction newFunction)
 {
-    parser.attach(command, (stringCallbackFunction)newFunction);
+  switch (command) {
```

Comment:
I just confirmed the binary is the same exact size, when using the single case `switch` or an `if` statement.